### PR TITLE
Introduce routines for manipulating with rules

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/issues/1403

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
Tested, ready to be reviewed.

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -73,9 +76,11 @@
 from .models import ExternalSoftwareEnvironment
 from .models import HardwareInformation
 from .models import InspectionRun
+from .models import KebechetGithubAppInstallations
 from .models import PackageExtractRun
 from .models import ProvenanceCheckerRun
 from .models import PythonArtifact
+from .models import PythonPackageVersionEntityRulesAssociation
```
Comment (harshad16) [inline thoth/storages/graph/postgres.py:83]: i see all the package have kept alphabetically ordered, this should go below.

**Review by harshad16 — COMMENTED** [review]
/lgtm
just one minor nit pick.
thanks :100:

File: thoth/storages/graph/postgres.py (hunk 1 above)
Comment (fridex) [reply thoth/storages/graph/postgres.py]: Thanks. Adjusted in a9d8629

**Pull request merged**